mapsims/noise.py

```diff
@@ -531,6 +547,102 @@ def get_white_noise_power(self, tube, sky_fraction, band=None, units="sr"):
             ret = ret[_band_index(tube, band)]
         return ret
 
+    def _get_wscale_factor(self, white_noise_rms, tube, sky_fraction):
+        """Internal function to re-scale white noise power
+        to a new value corresponding to white noise RMS in uK-arcmin.
+        """
+        if white_noise_rms is None:
+            return np.ones((2, 1))
+        cnoise = np.sqrt(
+            self.get_white_noise_power(tube, sky_fraction=1, units="arcmin2") * sky_fraction
```

Comment:
it seems to me that `_get_wscale_factor` and `get_white_noise_power` don't need to be part of the class, can you do standalone functions? so it is easier to understand what are inputs and outputs.

Reply:
Done for _get_wscale_factor, but get_white_noise_power does access class members.